common/xbps-src/shutils/chroot.sh

```diff
@@ -132,7 +132,10 @@ chroot_prepare() {
 
     # Prepare default locale: en_US.UTF-8.
     if [ -s ${XBPS_MASTERDIR}/etc/default/libc-locales ]; then
-        echo 'en_US.UTF-8 UTF-8' >> ${XBPS_MASTERDIR}/etc/default/libc-locales
+        printf '%s\n' \
```

Comment:
This should be two times `'%s\n%s\n'`, no?

Reply:
Bash specifies that

> The format is reused as necessary to consume all of the
arguments.  If the format requires more arguments than are
supplied, the extra format specifications behave as if a zero
value or null string, as appropriate, had been supplied.

Reply:
The argument still holds true for any POSIX conformant `printf(1)`.
Point 9. of EXTENDED DESCRIPTION https://pubs.opengroup.org/onlinepubs/9699919799/utilities/printf.html#tag_20_94_13

> The format operand shall be reused as often as necessary to satisfy the argument operands. Any extra b, c, or s conversion specifiers shall be evaluated as if a null string argument were supplied; other extra conversion specifications shall be evaluated as if a zero argument were supplied. If the format operand contains no conversion specifications and argument operands are present, the results are unspecified.